crates/fuel-core/Cargo.toml

```diff
@@ -72,12 +72,13 @@ mockall = { workspace = true }
 test-case = { workspace = true }
 
 [features]
-debug = ["fuel-core-types/debug"]
+dap = ["dep:uuid"]
+debug = ["fuel-core-types/debug", "dap"]
 default = ["debug", "metrics", "rocksdb"]
 metrics = ["dep:fuel-core-metrics"]
 p2p = ["dep:fuel-core-p2p", "dep:fuel-core-sync"]
 relayer = ["dep:fuel-core-relayer"]
 rocksdb = ["dep:rocksdb", "dep:tempfile"]
 test-helpers = ["fuel-core-p2p?/test-helpers"]
 # features to enable in production, but increase build times
-rocksdb-production = ["rocksdb?/jemalloc"]
+rocksdb-production = ["rocksdb", "rocksdb?/jemalloc"]
```

Comment:
nit: shouldn't this be just `"rocksdb/jemalloc"`, not conditonal on rocksdb since we explicitly enable it already?